.github/ISSUE_TEMPLATE/custom-pull-request-template.md

```diff
@@ -0,0 +1,30 @@
+# Description
+
+Please include a summary of the changes and attch the related issue link. 
+
+Summary: 
```

Comment:
It would be better if we make `Summary` and `Issue Link` bold, so like `**Summary**`, `Issue Link`

Reply:
sure, good idea, will do that